lib/aws_command.rb

```diff
@@ -7,7 +7,7 @@ module OpenShift
   module Ops
     class AwsCommand < Thor
       # WARNING: we do not currently support environments with hyphens in the name
-      SUPPORTED_ENVS = %w(prod stg int tint kint test jhonce amint tdint lint)
+      SUPPORTED_ENVS = %w(prod stg int tint kint test jhonce amint tdint lint jdetiber)
```

Comment:
What do you think of integrating the following hack ?

```
      SUPPORTED_ENVS = %w(prod stg int tint kint test jhonce amint tdint lint) << ENV['OO_CUSTOM_ENV']
```


This would allow the use of an environment variable named OO_CUSTOM_ENV, and would avoir that every one commits its one environment name.